docs/sdk/01-examples/02-multisig-client/10-get-voting-settings.md

```diff
@@ -21,17 +21,16 @@ const multisigClient: MultisigClient = new MultisigClient(contextPlugin);
 
 const daoAddressorEns: string = "0x12345348523485623984752394854320";
 
-const multisigSettings: MultisigVotingSettings = await multisigClient.methods.getVotingSettings(daoAddressorEns);
-console.log({ multisigSettings });
+const multisigVotingSettings: MultisigVotingSettings = await multisigClient
+  .methods.getVotingSettings(daoAddressorEns);
+console.log(multisigVotingSettings);
```

Comment:
Let's use the stanrdard ({ multisigVotingSettings }) so the console log is explicit about what it is returning.